app/views/questions/start_with_current_year/edit.html.erb

```diff
@@ -7,7 +7,7 @@
     </h1>
 
     <p>
-      <%=t("views.questions.start_with_current_year.info", :year => current_intake.most_recent_filing_year) %>
+      <%=t("views.questions.start_with_current_year.info", :year => current_intake.most_recent_needs_help_or_filing_year) %>
```

Comment:
do we want this logic change as part of this story?

Reply:
Yes - Asheesh